internal/pkg/global/config.go

```diff
@@ -445,6 +449,11 @@ func overloadFromEnv(c *AgentConfig) error {
 		c.Discovery.Docker.Regex = patterns
 	}
 
+	v = os.Getenv(strings.ToUpper(ConfigEnvPrefix + "_" + "runtime_ntp_server"))
+	if v != "" {
+		AgentConf.Runtime.NTPServer = v
+	}
+
```

Comment:
That's good. `ensureDefaults()` needs to also ensure `NTPServer` is always set to the default value.

For example:
```go
if c.Runtime.NTPSErver == "" {
    c.Runtime.NTPServer = DefaultNTPServer
}
```

Reply:
@iogakos Suggested changes has been made.

Reply:
@GuillaumeOuint For consistency, could you move the last three lines (default check) at the end of `ensureDefaults()`?